### DISCUSSION TITLE
求码

### DISCUSSION THREAD

**Discussion by shengweiice** [LFTEC/LearningFucker#32] [General]
------------------------------License------------------------------
Et07Ov0Nr3vYbuA7zj+eF6TpJ/zGWFJyBX0Ke2358gfawFFI/r30bAfq2w1joKqXGwn
ldYnqxZOhEQGSYYkoGYLucv7eqp2unQA1wZQreL2drdV8WH1FwKkmA7ONfMXMumR5WX
Skcq3nQeZ+j8hiyiA2YsdkEAfQ9s1R7Szytwvnu+CSSfx6AoqW69MV9ymxXmThbg1t5
Pjq2pcfl6sCOBSH19jVbjmwXYKE8HiHREieMUb4F91NQua/NYDxHgb3/zYeYnu2fxuU
Rjws/w4JNBTWUeOQvrDtX6hU5IX6cHuMm5SjtBH5WZg+8IhGRJWDgnqBcjCXae8CLUX
FR1QTvHY51lgotF2vR4BhiAE2xvZuI3ijQDIhoMZ2ivi92YvsGu4EJmjw6tR/GaVuXG
trLkx5P3ryyC00OjFQWXlg5PM2mN5ldZJKX8g8GyksJjkoGjLqoHSOTg3cgy7tauQ89
e+cor5NBPWDDHQ6j36I/Nq8TsPXBYAdLblWm/I+GWWwxUufZ+B6a5faSe4vAFKV9J+B
a2su/0fUFXpU6uB3fLSWiPFeqjSf8x+MEq2mDNuQKKH3xvmQOEcQIyycaYU9T4VzLkh
ArHj45P5j0MYbzxp+cb4L9kRJa7gPQHW7XM5u0jKIiic4vZdtD5tmyI6kceqIbHwrRJ
D2RSQl+LsgHKcvkeGHTdS4axtaOIoa41ZHvjVnJYIV38gBTepoX1DURnbI+Tmllf5I2
dBPIeMgfSmuDsbRnJekWB4EBX74faXe+1N3Zorx9ugs/Aj1Pe+2+TWwBfg1BV+k7L+U
QyRGCfNZWZ/LuxiNNY4i0iyRtmSwRL8OkF09dz3KUpxRUQmBhTwKWUkplc6wwVOSci6
kTo4aag7GU6xUjjLZ6ImI5f5Qnht1d9wNaI7ze4AoIVGkSaHvxpn+/71KHLrnPmuvUI
uxhrH416kIduGOF2qJaox9HKNIVToK693yrRM8oYh4YoHBC40acivwq72hTx/Jy3IWD
PUX3QBvrkFAw0mlRwwKW3g/On8vd4rzIH8PHHDJwH5w/ptjEvRj6EoKFvaiLtVqa1KI
w9XuWdwKwRKLadGu15VzpFZzofqNMjU4MyBcXnVhJw4zSqeOyY1bGNYMkApF3inCM0I
v0XWjIu07hs6UA/UJ5C55N2ZIoBDY7uk2RF+k3sC1lZEq6gayCVQ1AdecVz3aJZk5oK
5I7GessxB1Hh/4E7RHN+xiINacFqk/HqG/XReK4ttHB89is8SI4et8DCq95hLycGzao
pdZJcTh/enwa+lIi2IIGt5zTnuo5Y3+5xeL8YP1SWbbuJW0IEdwG3D9I+K9usyCtyK7
WLeX8tgCa/k4uQmdOJ0hwPRW/A==
------------------------------License------------------------------

**Comment by LFTEC** [comment]
kIN8SOGuTO249im1AJn8ENHbtCQGHKZwCyOk/JRi9oRbjvjKql1tuJe1tjqvUDlnVVvgs5MnkUaSQ8u5/M3qglPiO7w4FuNPmFV07ikCFI5wiCOiyhoJGyzixPcEvt4AL4AQpc0YxCF6vw/5ozuL1YWBqNmGTtCv+eRKzJrX8OjZszI0z+1f6/cvMgnB7iylz7hpqXV6cu99lbvESr59D4u7gKAn+08NODe0mSXlrY19lWJXHEWvvrFd2i8OoB0/uT5eHRqcil6bwl6cIIfbxfe2E1+mJocRTkOpXi9TZdqcXHt2RmEvBEQwcnXD9mTaZqEdAYdUADmmXuA91rT8P1iePpUZ7hBawvE7FzwXEaE9uYIO+55L5EiyZEN9a2vbCQ20lXbpKt5Ov8CQJiFM1mV9fdveMd7avgsA8n/g5fsaTz1fB5fUCGRv3AqazmY9XFGCaynZM92rgov6mN3RML2COq3yHr58houhWY0hWUutreJIgVHTbTm0ReiocljfcnhSO2oDqpOiEf3t/fG7VIj7vLTEBexYsCmr5Ki4QWqjJbhWVur9uBvkmJN2SHM8bAI+j2xBPcbHTO9UC6ijTLzpqM0zq4fYhi3xvasq8/0+sU/KZ4xuJTGbJLIlwwNNyqNvwQc+MhoaWL9PC6fHga5OGKJ/ksB6qDCw/AROfrNMxZDRPp9pm8pm34PeZeCkizBzBBj5USoX8J5NL9s4u2RRdC6pJC8ttkXf6qU/0gXQOmHmHdHlaj6CYE9PRvVsRDpvpFfSK2XtrD0t8sU7IOVxvhBijqi46fZjoOoTl07E7Bpa626Mxw+bh3wbUGnsjOmE0/tJ2d1+4se+dXrRVSfuHxOVa5Jlayo6xMMbMbbGkTqHgupJsr3Dq1/eZupHVo+wLLFoJog1sPTLDnoTYmwMnD8KGhGdpHaGtXK8GvjRbHd++5A/5CieD+aPOwQGTFPeXqqxob1vpAI9TcGHclTtfJk4gI6KWHbkw8A3ZXo7VOXK/zW3BgzWPwK8OR0Jco9Un9LegUYmW5gGixGYgiuXfFAZd+B+O4WOeREan8mc/MpfH/LhPwjOyJ0pAt69YtJ6BsQc81NB2vQ8S1ES82N5B/uZ1uhai49suKm42S7djSeMDM49q0jlHz5FoI6HHigdkL0F7MzwnoZidlqmd5YfHnR9n+p8rS2d9Z8kUcFGzv9fNkv9/g8l81bLF0EiSOBXAEIV/fTUUL3GzPXfy8ATvPAxjeWvIbPY4O8ZtLWU2ExYtbHPJRCuYeIhOC3Gf4MDwCe2X8slfP4OPyL9kPKXnSWlrrv41WkOj9qi8yCm0th7IwYZ3HuPvrb06vpNqR7eFEIBig1b4J/eHZiZdQ==
